src/js/controller/model.js

```diff
@@ -230,10 +236,15 @@ define([
             // Mute the video if autostarting on mobile. Otherwise, honor the model's mute value
             _provider.mute(this.autoStartOnMobile() || _this.get('mute'));
 
+            _provider.on('all', _videoEventHandler, this);
+
             // Attempt setting the playback rate to be the user selected value
             this.setPlaybackRate(this.get('defaultPlaybackRate'));
 
-            _provider.on('all', _videoEventHandler, this);
+            // Trigger ratechange because the provider may already have the rate in the above setPlaybackRate()
+            if (_provider.supportsPlaybackRate !== previousProviderPlaybackRateSupported) {
```

Comment:
Let's always set the `playbackRate` with a comment so we know why it's necessary.